src/cmd/src/cli/export.rs

```diff
@@ -70,6 +73,10 @@ pub struct ExportCommand {
     /// Things to export
     #[clap(long, short = 't', value_enum)]
     target: ExportTarget,
+
+    /// basic authentication for connecting to the server
+    #[clap(long)]
+    auth_basic: Option<String>,
```

Comment:
maybe move this to an upper-level command option in the future, if it's an common requirement